doc/string.md

```diff
@@ -45,21 +45,21 @@ Returns a UTF-16 encoded C++ string.
 
 ### New
 ```cpp
-String::New();
+Napi::String::New();
 ```
 
-Returns a new empty String
+Returns a new empty `Napi::String`.
 
 ### New
 ```cpp
-String::New(napi_env env, const std::string& value);
-String::New(napi_env env, const std::u16string& value);
-String::New(napi_env env, const char* value);
-String::New(napi_env env, const char16_t* value);
+Napi::String::New(napi_env env, const std::Napi::string& value);
+Napi::String::New(napi_env env, const std::u16Napi::string& value);
```

Comment:
I think these 2 might not be right as I don't expect std::Napi

Reply:
Ok just fixed!